instant-xml-macros/src/de.rs

```diff
@@ -0,0 +1,271 @@
+use proc_macro2::{Ident, Span, TokenStream};
+use quote::quote;
+
+use crate::{get_namespaces, retrieve_attr};
+
+struct Tokens<'a> {
+    enum_: &'a mut TokenStream,
+    consts_: &'a mut TokenStream,
```

Comment:
Why are all these names `_`-suffixed? I think that's only necessary for `enum` and `match`.